performant-software/digital-fauvel
# Move lists to language-code-named directories

### What does this do

- Moves all annotation lists to directories which match their corresponding language codes.
- Assigns ids to all annotations, based on language, folio and line.

### How to test

Confirm that this change doesn't break other parts of the editions (transcription viewer, etc).
